Sources/OpenCombineFoundation/Publishers+Timer.swift

```diff
@@ -0,0 +1,345 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+// Only support 64bit
+#if !(os(iOS) && (arch(i386) || arch(arm)))
```

Comment:
I don't see any reason to only support 64 bit systems. Is there a way to remove this?

To be clear, I understand that modern Apple operating systems are 64 bit-only, but we also support non-Apple platforms, and things may be different there.

Reply:
IDK, would probably need to write some tests and run them on 32-bit platforms to find out 😄

Reply:
We do run our tests on a 32 bit iOS simulator: https://app.circleci.com/pipelines/github/broadwaylamb/OpenCombine/373/workflows/e6c8e452-f56c-442d-8cc2-b7fceecdbe3c/jobs/1967/steps

Reply:
Yes, I'm aware of that, thanks 👍